src/rtc_base/platform_thread_types.cc

```diff
@@ -51,7 +53,7 @@ PlatformThreadId CurrentThreadId() {
 #elif defined(__EMSCRIPTEN__)
   return static_cast<PlatformThreadId>(pthread_self());
 #else
-  // Default implementation for nacl and solaris.
+  // Default implementation for nacl, openbsd and solaris.
```

Comment:
We don't want to differ too much with upstream, no unneeded modifications please